src/js/components/Ballot/BallotTitleHeaderNationalPlaceholder.jsx

```diff
@@ -83,6 +85,27 @@ class BallotTitleHeaderNationalPlaceholder extends Component {
       const showEditAddress = true;
       const showSelectBallotModal = true;
       // this.props.toggleSelectBallotModal('', showEditAddress, false);
+      const { location: { pathname: currentPathname } } = window;
+      const page = lookupPageNameAndPageTypeDict(currentPathname);
+      const dataLayerObject = {
+        event: 'action',
```

Comment:
Please make sure to add an `actionDetails` dictionary above event. See the Property Name spreadsheet (see link below) for more information.